knotx-core/src/test/java/io/knotx/fragments/FragmentContentExtractorTest.java

```diff
@@ -37,8 +42,22 @@
   public void getUnwrappedContent_withFragment_expectDefinedContent(Fragment fragment,
```

Comment:
Looking at the test input, it seems we're only testing the happy paths here. How is the class under test supposed to behave on invalid input?